service/src/cli.yml

```diff
@@ -57,6 +57,10 @@ subcommands:
                 required: false
                 index: 1
                 help: shard identifier base58 encoded. Defines the state that this worker shall operate on. Default is mrenclave
+            - dev:
+                long: dev
+                short: d
+                help: Alice is faucet for development purposes. Bootstrap enclave funds from //Alice account.
```

Comment:
```suggestion
                help: Bootstrap enclave account on parentchain via //Alice.
```

Reply:
I miss a note about development. 
help: Set this flag if running in development mode to bootstrap enclave account on parentchain via //Alice.